documentation/asciidoc/accessories/display/legacy.adoc

```diff
@@ -1,6 +1,31 @@
 == Legacy Support
 
-NOTE: These instructions are for the original Raspberry Pi 1 Model A and B boards only. You can identify an original board as it is the only model with a 26-pin GPIO header, all other models have the now-standard 40-pin connector.
+[NOTE]
+====
+Older Raspberry Pi hardware, and older versions of Raspberry Pi OS, with X11 can install an on-screen keyboard and emulate right clicks on the touchscreen. 
+
+Run the following in a terminal to enable the keyboard:
```

Comment:
"enable the keyboard" -> "install the on-screen keyboard"

Reply:
Yep, should've been more consistent there; fixed